src/Issues.js

```diff
@@ -1,11 +1,15 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import IssueList from "./IssueList";
 import { Link } from "react-router-dom";
 
 export default function Issues({ issues, labels, labelsById }) {
-  const [selectedLabelId, setSelectedLabelId] = useState();
+  const [selectedLabelId, setSelectedLabelId] = useState("-");
   const [filteredIssues, setFilteredIssues] = useState(issues);
 
+  useEffect(() => {
+    setFilteredIssues(issues);
+  }, [issues]);
```

Comment:
Whenever the `issues` prop changes, update `filteredIssues`. Without this, the issues won't render because the `issues` prop in this component will initially be an empty array, resulting in `filteredIssues` being an empty array, and `filteredIssues` will never get updated.